### DISCUSSION TITLE
"Last modified" button should show the last modified first

### DISCUSSION THREAD

**Discussion by edhelas** [jcorporation/myMPD#1004] [Ideas]
myMPD version: 10.2.5

A simple request regarding the UX logic. 
The "Last modified" button should actually order the albums by last-modified first (and not the way around)

![image](https://user-images.githubusercontent.com/3726886/226107730-338e5125-fdf6-41ad-b468-bfe604c54a9c.png)

When I'm ordering the albums this way I always check the "Desc" checkbox just bellow.

**Comment by jcorporation** [comment]
But then the descending check box is misleading.

**Reply by GK-774** [reply]
Well, it would need to be renamed to "ascending" then.

**Reply by GK-774** [reply]
I always use "descending" on (and have a home icon that does that), and find to see the newest albums first more useful. Actually "last modified" would mean to see the albums last modified (the newest) first, IMHO, so "descending" as default would be more logical.

**Reply by jcorporation** [reply]
For all other tags ascending is the best order. 

Renaming Descending to Reverse could be an idea.

**Reply by edhelas** [reply]
To me it is the way I'm also using it. I'm adding new albums in my audio library and I like to browse it with "more recent" first and go back through time this way :)

**Comment by GK-774** [comment]
I understand the argument with the other tags, sorting in numerical or alphabetical order. But I think "last modified" would mean "zuletzt geändert" (If I´m correct, I´m not a native English speaker) and this would imply "newest first". Currently it rather sorts by "modification date" not "last modified". But then, certainly a very minor "issue".

**Reply by jcorporation** [reply]
Currently myMPD reflects the MPD behavior. I am unsure, if we should change this, but I also want the most time to see the newest first.

**Comment by jcorporation** [comment]
This is implemented for v10.3.0
